bucket/virt-viewer.json

```diff
@@ -0,0 +1,41 @@
+{
+    "version": "10.0-1.0",
+    "description": "Open source tool to connect to virtual machines to display VNC or SPICE",
+    "homepage": "https://virt-manager.org/download/",
+    "license": "GPL v3+",
+    "architecture": {
+        "64bit": {
+            "url": "https://virt-manager.org/download/sources/virt-viewer/virt-viewer-x64-10.0-1.0.msi",
+            "hash": "f50c13602060184b3e2db5f5822ff1860ac2eec39660578774008892e3516166"
+        },
+        "32bit": {
+            "url": "https://virt-manager.org/download/sources/virt-viewer/virt-viewer-x86-10.0-1.0.msi",
+            "hash": "ada7d13136f8ec2cf574071aa0bc1c1b8cf14ee62614b0728cdce5a46b080934"
+        }
+    },
+    "extract_dir": "VirtViewer v10.0-256",
+    "bin": [
+        "bin\\virt-viewer.exe",
+        "bin\\remote-viewer.exe"
+    ],
+    "shortcuts": [
+        [
+            "bin\\remote-viewer.exe",
+            "Remote Viewer"
+        ]
+    ],
+    "checkver": {
+        "url": "https://virt-manager.org/download/",
+        "regex": "virt-viewer-x64-([\\d.-]+)\\.msi"
+    },
+    "autoupdate": {
```

Comment:
Autoupdate does not make sense until extract dir is changed

Reply:
Is there a way to automatically the extract_dir of VirtViewer v10.0-*? Or should I just delete the autoupdate section?

Reply:
They have replied: "This value 256 reflects the way we encode the buildid ("-1.0") part of the version number. Windows only permits a single value in this context, but we need two values, so we have to encode them. A future "-1.1" would be encoded to 257, and "-2.0" would become 512, and so on."

I have found a way to convert the 256 to 1.0 in order to update the extract_dir. So I have removed the deleted the autoupdate section in the manifest. If anyone have a way to convert it please help me edit it.